index.html

```diff
@@ -62,6 +62,157 @@ <h1>
 
         </div>
     </section>
+    <!-- Works -->
+    <section class="works">
+        <div class="works_content">
+            <h1>My Recent Works</h1>
```

Comment:
- [ ] While I appreciate🎉🎉 your efforts to include `My Recent Works,` it seems that it is not present in the Figma design. 
- [ ] I kindly🙏🙏 suggest that you follow the suggested [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=xw0Xo7v32ZLKEBS9-0) design closely. 
- [ ] Additionally, please keep in mind that for better `SEO` ranking, it is recommended to have only one `<h1>` tag per HTML page.  For more details please follow this [link](https://www.semrush.com/blog/h1-tag/) 🤔🤔

Reply:
@AbdusSattar-70 I really appreciate your review but the title "My Recent Works" it is present in Figma template you may please double-check it. Thank you